chat-plugins/tcg.js

```diff
@@ -20,7 +20,13 @@ var tcgsearch = function (target, room, user, cmd, self) {
 			response += data;
 		});
 		a.on('end', function () {
-			var result = JSON.parse(response);
+			var result;
+
+			try {
+				result = JSON.parse(response);
+			} catch (e) {
+				return self.sendReply("ERROR: Could not parse query:" + e);
```

Comment:
If I put in a command and it doesn't work, I would want to know it doesn't work and what the error is. Idk I can change it easily.